vowpalwabbit/array_parameters.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+#include <stdio.h>
+#include <iostream>
+#include <stdlib.h>
+#include <string.h>
+
+using namespace std;
+typedef float weight;
+
+class array_parameters
+{
+private:
+	weight* _begin;
+
```

Comment:
You probably want all the elements here:
https://github.com/JohnLangford/vowpal_wabbit/blob/master/vowpalwabbit/global_data.h#L117 folded into this class.